### DISCUSSION TITLE
is there syntax change between v2 and v3?

### DISCUSSION THREAD

**Discussion by trim21** [beancount/beancount#866] [Q&A]
is there syntax change between v2 and v3?

**Comment by blais** [comment]
v2 branch is dead. Neither remember nor care.

Diffing the lexer and grammar files is helpful:

https://github.com/beancount/beancount/compare/v2...v3?diff=unified&w=#diff-0ebd682b93af8679f7037952e2687f178ceaa16dd1d5718f820fd7aaf7329d72

https://github.com/beancount/beancount/compare/v2...v3?diff=unified&w=#diff-61912f791c5fc22ed0b4fb3e3490e786a8f45a964f7a368e70450e84154f019c


(IIRC there are little changes.)

**Comment by trim21** [comment]
looks like beancount/parser/lexer.l doesn't change much

**Reply by blais** [reply]
Yeah, it's all that very different.

**Reply by trim21** [reply]
I'm not native English speaker, do you mean there are many differences or there are not many differences?

**Comment by blais** [comment]
Not many differences.
